packages/regl-worldview/src/stories/assertionStories/Worldview.stories.js

```diff
@@ -175,4 +184,54 @@ stories
         expect(result[0].object).toEqual(cube);
       },
     })
+  )
+  .add(
+    `a component's mouse handlers can stop event propagation to worldview's global mouse handlers`,
+    assertionTest({
+      story: (setTestData) => (
+        <WorldviewWrapper onMouseDown={(_, { objects }) => setTestData([])}>
+          <Cubes
+            onMouseDown={(e, { objects }) => {
+              e.stopPropagation();
+              setTestData(objects);
+            }}>
+            {[cube]}
+          </Cubes>
+        </WorldviewWrapper>
+      ),
+      assertions: async (getTestData) => {
+        await emitMouseEvent("mousedown");
+        const result = await getTestData();
+        expect(result.length).toEqual(1);
+        expect(result[0].object).toEqual(cube);
+      },
+    })
+  )
+  .add(
```

Comment:
Would be nice to add more tests with different `layerIndex` and order , e.g. 
```
<Cube layerIndex={2} {... stopPropgation()}><Cubes>
<Cube layerIndex={1}><Cubes>
```
or:
```
<Cube layerIndex={1}><Cubes>
<Cube layerIndex={2} {... stopPropgation()} ><Cubes>
```